dev/create-release/spark-rm/Dockerfile

```diff
@@ -15,16 +15,20 @@
 # limitations under the License.
 #
 
-# Image for building Spark releases. Based on Ubuntu 18.04.
+# Image for building Spark releases. Based on Ubuntu 20.04.
 #
 # Includes:
 # * Java 8
 # * Ivy
-# * Python (2.7.15/3.6.7)
-# * R-base/R-base-dev (4.0.2)
-# * Ruby 2.3 build utilities
+# * Python (3.8.5)
```

Comment:
Does the doc generation work with Python 3.8, @HyukjinKwon ?

Reply:
Yeah, I use Python 3.8 locally :-). I will happen to double check when I do actual release.
BTW, after we switch to Rouge at https://github.com/apache/spark/pull/26521, we're not dependent on Python anymore in other documentation generations if I am not mistaken.